typescript/cli/src/commands/core.ts

```diff
@@ -146,7 +145,7 @@ export const read: CommandModuleWithContext<{
       const coreConfig = await evmCoreReader.deriveCoreConfig(mailbox);
       writeYamlOrJson(configFilePath, coreConfig, 'yaml');
       logGreen(`✅ Core config written successfully to ${configFilePath}:\n`);
-      log(indentYamlOrJson(yamlStringify(coreConfig, null, 2), 4));
+      logYamlIfUnderMaxLines(coreConfig, 250);
```

Comment:
store 250 as a const like `MAX_READ_LINE_OUTPUT` to be shared between core and warp